examples/createOrder.ts

```diff
@@ -0,0 +1,33 @@
+import { ImmutableX, Config, UnsignedOrderRequest } from '@imtbl/core-sdk';
+import { generateWalletConnection } from './libs/walletConnection';
+
+(async () => {
+  const walletConnection = await generateWalletConnection('goerli');
+
+  const imxClient = new ImmutableX(Config.SANDBOX);
+
+  const orderParams: UnsignedOrderRequest = {
+    buy: {
+      type: 'ETH',
+      amount: '', // amount to buy the NFT (asset)
+    },
+    sell: {
+      type: 'ERC721',
+      tokenAddress: '',
+      tokenId: '',
```

Comment:
Should be able to rely on intellisense here for descriptions. Is there a need for the comment? If so, can we comment the others?

Reply:
Yes it's all on intellisense. Removed comments.
[a7f99be](https://github.com/immutable/imx-core-sdk/pull/195/commits/a7f99bee29bf808011d4355d4e30590187b35fcc)